Content.Client/Weapons/Marker/ClientDamageMarkerSystem.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Shared.Weapons.Marker;
+using Robust.Client.GameObjects;
+using Robust.Shared.Timing;
+
+namespace Content.Client.Weapons.Marker;
+
+public sealed class ClientDamageMarkerSystem : EntitySystem
```

Comment:
shouldnt this inherit the shared system (and dont be called Clientsomethingsystem)